test/test_biogeochemistry.jl

```diff
@@ -68,7 +68,7 @@ end
 const MB = Union{MinimalDiscreteBiogeochemistry, MinimalContinuousBiogeochemistry}
 
 @inline          required_biogeochemical_tracers(::MB) = tuple(:P)
-@inline required_biogeochemical_auxiliary_fields(::MB) = tuple(:Iᴾᴬᴿ)
+@inline required_biogeochemical_auxiliary_fields(::MB) = ()
```

Comment:
We want this for the test?

Reply:
It seems that I^{PAR} is repeated in fields. So, indeed, it looks like the `bialogical_auxiliary_fields` are added to the model's auxiliary fields upon model construction here:
```julia
    biogeochemical_fields = merge(auxiliary_fields, biogeochemical_auxiliary_fields(biogeochemistry))
    tracers, auxiliary_fields = validate_biogeochemistry(tracers, biogeochemical_fields, biogeochemistry, grid, clock)
```
which leads to having repeated I^{PAR} here
```julia
@inline fields(model::HydrostaticFreeSurfaceModel) = 
    merge(hydrostatic_fields(model.velocities, model.free_surface, model.tracers), 
          model.auxiliary_fields,
          biogeochemical_auxiliary_fields(model.biogeochemistry))
```
So I think we have a decision to make
(1) leave the biogeochemical fields in the `model.biogeochemistry`
(2) copy them in `model.auxiliary_fields` but modify the `fields` function

Reply:
`required_biogeochemical_auxiliary_fields` are the `model.auxiliary_fields`. Then there are also `biogeochemical_auxiliary_fields(model.biogeochemistry)`, which are distinct? So there are two lists of auxiliary fields.

We can change the design.